Gems/Multiplayer/Code/Source/Components/NetworkHitVolumesComponent.cpp

```diff
@@ -125,9 +125,18 @@ namespace Multiplayer
         EMotionFX::Integration::ActorComponentNotificationBus::Handler::BusConnect(GetEntityId());
         GetNetBindComponent()->AddEntitySyncRewindEventHandler(m_syncRewindHandler);
         GetNetBindComponent()->AddEntityPreRenderEventHandler(m_preRenderHandler);
-        m_physicsCharacter = Physics::CharacterRequestBus::FindFirstHandler(GetEntityId());
         GetTransformComponent()->BindTransformChangedEventHandler(m_transformChangedHandler);
         OnTransformUpdate(GetTransformComponent()->GetWorldTM());
+
+        // During activation the character controller is not created yet.
+        // Connect to CharacterNotificationBus to listen when it's activated after creation.
+        Physics::CharacterNotificationBus::Handler::BusConnect(GetEntityId());
```

Comment:
Missing to disconnect from the bus in `:OnDeactivate` (even if it's done in `OnCharacterActivated` because if this component is deactivated before that's call you'd still need to disconnect from the bus).

Reply:
Done